recipes/slimit/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set name = "slimit" %}
+{% set version = "0.8.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://github.com/rspivak/{{ name }}/archive/{{ version }}.tar.gz"
+  sha256: "015fc1917c0eff834514fcedd4251275a3dacb10df1b49549277fe5e19de1ddf"
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
```

Comment:
Can this package be `noarch: python`? See the documentation for more information: https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-python

Reply:
This package uses 2to3, so I don't believe so